.ruby-version

```diff
@@ -1 +1 @@
-ruby-3.1.2
+ruby-3.2.2
```

Comment:
Fix: ruby versions from this file and `Gemfile.lock` were not the same, causing issues on installation.